[14.0]account_payment_order - hide reference_type from invoice view

luc-demeyer opened the pull request:
I think we should not add the reference_type field for outgoing invoices in the account_payment_order module.
Adding this field for outgoing invoices adds confusion for the end user.
It could be added by modules that introduce other payment reference types, but even in that case I am not sure we need to visualise this field on outgoing invoices since the structured communication will probably be autocalculated and readonly for outgoing invoices.

Murtaza-OSI committed:
[ADD] account_payment_order_vendor_email

ao-landoo committed:
[FIX] Post mandate expiration message by register

etobella committed:
[FIX] account_banking_sepa_direct_debit: Bank Account and Mandate should not be required

The bank account is taken from the mandate. Mandate is autopopulated if not found.

etobella committed:
[FIX] account_banking_sepa_credit_transfer: Bank account is not required

luc-demeyer committed:
[14.0]hide reference_type from invoice view

luc-demeyer committed:
[14.0]hide reference_type from invoice view

pedrobaeza commented:
I don't use such field, but not sure if for Belgium with BBA is needed. @sbidoul do you use it?

pedrobaeza commented:
@luc-demeyer remember to put the involved module in the commit title as well on the PR title.

OCA-git-bot committed:
[UPD] addons table in README.md

alexis-via committed:
sepa_direct_debit: mandate_required back to True

sepa_credit_transfer: bank_account_required back to True
Partial revert of the breakage of PR #875

OCA-git-bot committed:
Merge PR #888 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_banking_sepa_direct_debit 14.0.1.3.0

OCA-git-bot committed:
account_banking_sepa_credit_transfer 14.0.1.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

Yvesldff committed:
Translated using Weblate (French)

Currently translated at 70.7% (121 of 171 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_banking_pain_base
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_banking_pain_base/fr/

celm1990 committed:
[IMP] account_payment_order: outbound inheritable + SavepointCase
This commit split the tests for outbound payment in a base inheritable class that
can be reused in other dependant modules, and the test of this module.

similar to https://github.com/OCA/bank-payment/commit/86bd1a2525d45f4a2eb9146b4a0ef679068d3649

pedrobaeza committed:
[FIX] account_payment_order: Proper indentation for test + make it work

The test introduced in 7bdb286aa1a3d5beb8ee95193ed6eb4abc67ce1e is not
valid, as its indentation made it to not be executed. Anyways, the
content and the tested things are also not correct.

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 55.2% (21 of 38 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_partner
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_partner/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 92.1% (35 of 38 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_partner
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_partner/it/

SicurSam committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (38 of 38 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_partner
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_partner/it/

olgamarcocb committed:
Translated using Weblate (Spanish)

Currently translated at 80.0% (4 of 5 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_sale
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_sale/es/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 86.8% (33 of 38 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_partner
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_partner/nl/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 90.3% (47 of 52 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_mode
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_mode/nl/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 69.8% (148 of 212 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_order
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_order/sv/

simonstromb committed:
Added translation using Weblate (Swedish)

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (17 of 17 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_banking_sepa_credit_transfer
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_banking_sepa_credit_transfer/sv/

simonstromb committed:
Added translation using Weblate (Swedish)

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 2.2% (4 of 174 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_banking_pain_base
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_banking_pain_base/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 72.1% (153 of 212 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_order
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_order/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 98.5% (209 of 212 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_order
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_order/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 2.8% (5 of 174 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_banking_pain_base
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_banking_pain_base/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 92.1% (35 of 38 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_partner
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_partner/sv/

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (8 of 8 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_purchase
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_purchase/sv/

simonstromb committed:
Added translation using Weblate (Swedish)

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (4 of 4 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_purchase_stock
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_purchase_stock/sv/

alexis-via committed:
account_payment_order: communication required in form view

OCA-git-bot committed:
Merge PR #897 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_payment_order 14.0.1.5.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #776 into 14.0

Signed-off-by dreispt

oca-travis committed:
[UPD] Update account_payment_order_vendor_email.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

Shide committed:
[FIX] [14.0] Removed external dependencies already defined in Odoo base

OCA-git-bot committed:
Merge PR #901 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_banking_sepa_direct_debit 14.0.1.3.1

OCA-git-bot committed:
[UPD] addons table in README.md

MiquelRForgeFlow committed:
[IMP] account_payment_order: don't requiere reference_type anymore

MiquelRForgeFlow committed:
[FIX] account_payment_partner: only use bank_account_required in bills

MiquelRForgeFlow committed:
[FIX] account_banking_mandate: only use mandate_required in invoices

OCA-git-bot committed:
Merge PR #908 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_banking_mandate 14.0.1.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #903 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_payment_order 14.0.1.6.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #905 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_payment_partner 14.0.1.4.0

OCA-git-bot committed:
[UPD] addons table in README.md

simonstromb committed:
Translated using Weblate (Swedish)

Currently translated at 98.5% (209 of 212 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_order
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_order/sv/

OCA-git-bot committed:
[IMP] update dotfiles [ci skip]

victoralmau committed:
[FIX] account_banking_sepa_direct_debit: Remove warnings in tests.
TT35948

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 50.0% (4 of 8 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_purchase
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_purchase/it/

OCA-git-bot committed:
Merge PR #917 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_banking_sepa_direct_debit 14.0.1.3.2

OCA-git-bot committed:
[UPD] addons table in README.md

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (8 of 8 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_banking_mandate_sale
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_banking_mandate_sale/ca/

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (212 of 212 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_order
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_order/ca/

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (8 of 8 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_purchase
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_purchase/ca/

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (7 of 7 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_sale
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_sale/ca/

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (38 of 38 strings)

Translation: bank-payment-14.0/bank-payment-14.0-account_payment_partner
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-14-0/bank-payment-14-0-account_payment_partner/ca/

rousseldenis committed:
[14.0][IMP] account_payment_order: Add a hook method to retrieve communication type and communication

rousseldenis committed:
[14.0][IMP] account_payment_order: Improve normal communication if there is a credit note

If there is a credit note that partially cancel an invoice, the payment communication
should be the combination of the invoice reference and the credit note one.

rousseldenis committed:
[14.0][IMP] account_payment_order: Remove not needed assert as parameter is required

rousseldenis committed:
[14.0][IMP] account_payment_order: Use the 'payment_reference' field if filled in

rousseldenis committed:
[14.0][IMP] account_payment_order: Set tests as post-install

Since https://github.com/odoo/odoo/commit/81aac30dd2278e43a43cf8fd6cef31e1e8c60f3f
AccountTestInvoicingCommon should be tagged post-install

rousseldenis commented:
@luc-demeyer

OCA-git-bot committed:
Merge PR #918 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
account_payment_order 14.0.1.7.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #874 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_banking_sepa_direct_debit 14.0.1.3.3

OCA-git-bot committed:
[UPD] addons table in README.md

luc-demeyer committed:
Merge branch '14-hide-reference_type' of github.com:luc-demeyer/bank-payment into 14-hide-reference_type
